Ports/SuperTuxKart/patches/0005-Correct-the-resource-install-path.patch

```diff
@@ -0,0 +1,22 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Dominika Liberda <ja@sdomi.pl>
+Date: Sun, 2 Jul 2023 07:21:46 +0200
+Subject: [PATCH] Correct the resource install path
+
+---
+ CMakeLists.txt | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index a68f38ec32e7d240dae28998caf5511e936533ea..38a93e7bf5bd388e9b08810cd2488ba8fefff711 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -593,7 +593,7 @@ else()
+     if(NOT STK_INSTALL_DATA_DIR_ABSOLUTE)
+         get_filename_component(STK_INSTALL_DATA_DIR_ABSOLUTE ${STK_INSTALL_DATA_DIR} ABSOLUTE)
+         if(${STK_INSTALL_DATA_DIR_ABSOLUTE} STREQUAL ${STK_INSTALL_DATA_DIR})
+-            add_definitions(-DSUPERTUXKART_DATADIR=\"${STK_INSTALL_DATA_DIR_ABSOLUTE}\")
++            add_definitions(-DSUPERTUXKART_DATADIR=\"/opt/SuperTuxKart\")
```

Comment:
If you want the install data dir to be /opt/SuperTuxCart instead of CMAKE_INSTALL_PREFIX/CMAKE_INSTALL_DATA_DIR (which will end up being Build/x86_64/Root/opt/SuperTuxCart/share/supertuxcart by default), then set the option from line 141 on the command line via configopts instead of patching it here.

https://github.com/supertuxkart/stk-code/blob/1.4/CMakeLists.txt#L141

Personally, I would expect the else case to do the right thing, especially since we'll have CMAKE_STAGING_SYSROOT or w/e set in the CMakeToolchain.txt to not muddy CMAKE_INSTALL_PREFIX with host paths.

Reply:
Doesn't look like this flag has fixed it, sadly. I've tried basically all flags for changing the directory *before* resorting to patching stuff :(

Reply:
Hmm all right. Setting the flag worked for me in Ubuntu but it's possible there's some extra quirks going on